usage/launcher/src/main/java/brooklyn/launcher/BrooklynLauncher.java

```diff
@@ -772,6 +782,7 @@ protected void startWebApps() {
             webServer.setPublicAddress(publicAddress);
             if (port!=null) webServer.setPort(port);
             if (useHttps!=null) webServer.setHttpsEnabled(useHttps);
+            webServer.setShutdownRequestListener(shutdownRequestListener);
```

Comment:
I was initilly a little worried about nulls being passed through here (but looks like it's ok at the end, where it's finally used).

If nulls are allowed, then worth including `@Nullable` on the parameter.